backend/db/migrations/20240312000713-create-user.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+let options = {};
+if (process.env.NODE_ENV === 'production') {
+  options.schema = process.env.SCHEMA; // define your schema in options object
+}
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable(
+      'Users',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        firstName: {
+          type: Sequelize.STRING(30),
+          allowNull: false,
+        },
+        lastName: {
+          type: Sequelize.STRING(30),
+          allowNull: false,
+        },
+        email: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        username: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
+        },
+      },
+      options
+    );
+  },
+  async down(queryInterface, Sequelize) {
+    await queryInterface.dropTable('Users');
```

Comment:
You need to make sure you are including your options object in your drop statement as well. Take a look at the code shown in phase 3.

![image](https://github.com/bnaluz/nestBnb/assets/92609467/de7af4dd-36a9-405b-b1d9-75b5caf119b7)

That options object lets your render application know which schema to delete from. All of your projects deployed on render share the same database, so when you are migrating you need to let render know which schema you are migrating to.

Reply:
i updated the migration file, is it reflecting on your end?

Reply:
looks good. feel free to merge.